.gitignore

```diff
@@ -475,3 +475,7 @@
 
 # /utils/wxrc/
 /utils/wxrc/wxrc_vc[789].sln
+
+#Ignore OS Specific files
+#MAC OS
+.DS_Store
```

Comment:
Please don't do this in wx `.gitignore`, it should be done once for all projects in the global one and not here.

Reply:
can you  explain where exactly is the global ignore? 
I thought this was the right place!

Reply:
The value of `core.excludesfile` option, by default `~/.gitignore`.

Reply:
Ok noted, thank you!